x-pack/plugins/screenshotting/server/browsers/chromium/driver_factory/index.ts

```diff
@@ -312,7 +312,7 @@ export class HeadlessChromiumDriverFactory {
   }
 
   getBrowserLogger(page: Page, logger: Logger): Rx.Observable<void> {
-    const consoleMessages$ = Rx.fromEvent<ConsoleMessage>(page, 'console').pipe(
+    const consoleMessages$ = Rx.fromEvent<ConsoleMessage>(page as any, 'console').pipe(
```

Comment:
@elastic/kibana-reporting-services the add/removeEvelentListenner were removed from puppeteer at https://github.com/puppeteer/puppeteer/pull/11792 .

Could you please help us verify if that has any real effect on the final purpose of your code here? (Same thing on the 3 other lines changed on that PR below the current one)